external-api/api-tests/v1/eStore.api-test.ts

```diff
@@ -36,17 +36,17 @@ const mock = new MockAdapter(axios);
 jest.mock('axios', () => jest.requireActual('axios'));
 
 const mockResponse = {
-  siteName: 'test',
+  siteId: 'test',
   status: 'Created',
 };
 
 // mocks test for estore if exists
-mock.onPost(`${eStoreUrl}/site/exist`).reply(200, mockResponse);
+mock.onPost(`${APIM_ESTORE_URL}/site/sites?exporterName=testName`).reply(200, mockResponse);
 
 describe('/estore', () => {
   const payload = {
     dealId: '12345',
-    siteName: 'google',
+    siteId: 'google',
```

Comment:
Can we change `google` to `ukef` please.

Reply:
updated now